lib/mix/lib/mix/tasks/test.ex

```diff
@@ -153,15 +153,16 @@ defmodule Mix.Tasks.Test do
     * `--exclude` - excludes tests that match the filter
     * `--failed` - runs only tests that failed the last time they ran
     * `--force` - forces compilation regardless of modification times
-    * `--formatter` - formatter module
+    * `--formatter` - sets the formatter modules that will print the results.
```

Comment:
I'll keep the singular here even if you can pass the `--formatter` option more than once, because with the plural it seems you should pass a list.

```suggestion
    * `--formatter` - sets the formatter module that will print the results.
```

Reply:
I was in doubt when I edited this. Thank you